mysqld_exporter.go

```diff
@@ -69,6 +71,7 @@ const (
 	informationSchema = "info_schema"
 	performanceSchema = "perf_schema"
 	slaveStatus       = "slave_status"
+	database		  = "database"
```

Comment:
Also, I think this should be `tableStatus = "table_status"` since the data comes from `SHOW TABLE STATUS`